dev/check-license

```diff
@@ -71,6 +72,11 @@ mkdir -p "$FWDIR"/lib
 
 $java_cmd -jar "$rat_jar" -E "$FWDIR"/.rat-excludes  -d "$FWDIR" > rat-results.txt
 
+if [ $? -ne 0 ]; then
+   echo "Error happen when running rat, pls checking the rat jar"
```

Comment:
This seems fine except that this message needs improvement. 'RAT exited abnormally' is probably fine.